README.md

```diff
@@ -46,25 +35,16 @@ https://github.com/rmcolq/make_prg for a pipeline which can construct
 these PRGs from a set of aligned sequence files.
 
 Pandora can do the following for a single sample (read dataset):
-- Output inferred mosaic of reference sequences for loci (eg genes) from
-  the PanRG which are present
-- Output a VCF showing the variation found within these loci, with
-  respect to any reference path in the PRG.
-
-Soon, in a galaxy not so far away, it will allow:
-- discovery of new variation not in the PRG
+- Output inferred mosaic of reference sequences for loci (eg genes) from the PanRG which are present;
+- Output a VCF showing the variation found within these loci, with respect to any reference path in the PRG;
+- Discovery of new variation not in the PRG.
 
 For a collection of samples, it can:
-- Output a matrix showing inferred copy-number of each locus in each
-  sample genome
-- Output a multisample pangenome VCF showing how including genotype
-  calls for each sample in each of the loci
-- Output one VCF per orthologous-chunk, showing how samples which
-  contained this chunk differed in their gene sequence. Variation is
-  shown with respect to the most informative recombinant path in the
-  PRG.
+- Output a matrix showing inferred copy-number of each locus in each sample genome;
+- Output a multisample pangenome VCF showing how including genotype calls for each sample in each of the loci;
```

Comment:
This sentence seems incomplete?

Reply:
It does indeed, this was before in the README... will fix this